docs/email-someone-if-shipping-address-fsa-is-out-of-delivery-area/script.liquid

```diff
@@ -0,0 +1,33 @@
+{% assign available_fsa = options.allowed_fsa__required | split: "," %}
+{% assign order_fsa = order.shipping_address.zip | slice: 0, 3 %}
+
+{% if available_fsa contains order_fsa %}
+	{% assign order_disqualified = false %}
```

Comment:
Use consistent, two-space indenting. You've got some lines indented with tab characters - make sure those are replaced with two spaces each.